pkg/common/providers/ocmprovider/cluster.go

```diff
@@ -236,7 +236,18 @@ func (o *OCMProvider) DeleteCluster(clusterID string) error {
 	if err != nil {
 		return fmt.Errorf("couldn't delete cluster '%s': %v", clusterID, err)
 	}
-	return nil
+
+	time.Sleep(300 * time.Second)
```

Comment:
We shouldn't block here. What would be more appropriate would be to use a `wait.PollImmediate`. See https://github.com/openshift/osde2e/blob/5c18d4991183b47c7e34e8755789947fd4e5f8b7/pkg/e2e/verify/namespace_webhook.go#L280-L284 for an example

Reply:
Also I'd poll ever 15s over 5min